style.css

```diff
@@ -525,3 +525,562 @@ img:hover {
+    display: flex;
+    flex-direction: row-reverse;
+    width: 39.5%;
+    background-color: #1c1a19;
+    align-items: center;
+    justify-content: center;
+  }
+
+  .header-intro {
+    margin-left: 10%;
+  }
+
+  .header-text {
+    font-size: 72px;
+    line-height: 113%;
+    color: #ff6b00;
+    margin-block-start: 0;
+    margin-block-end: 0;
+    margin-left: 14px;
+    margin-top: 16rem;
+  }
+
+  .header-smaller-text {
+    font-family: 'Inter', sans-serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 35px;
+    margin-block-start: 0;
+    margin-block-end: 0;
+    line-height: 44px;
+    padding-top: 2px;
+    margin-top: 0;
+  }
+
+  .header-paragraph {
+    font-family: 'Inter', sans-serif;
+    font-size: 20px;
+    line-height: 32px;
+    margin-bottom: 15.6rem;
+  }
+
+  .header-icons {
+    display: flex;
+    padding-left: 10px;
+  }
+
+  .header-icons ul {
+    display: flex;
+    flex-direction: column;
+    padding-bottom: 0;
+  }
+
+  .header-icons ul li {
+    padding-left: 20px;
+    padding-bottom: 10px;
+  }
+
+  .work-section {
+    display: flex;
+    flex-direction: column;
+  }
+
+  .workSection1 {
+    display: flex;
+    flex-direction: row;
+    width: 63.6%;
+  }
+
+  .work-section-heading h2 {
+    padding: 25px 0 19px 0;
+    font-weight: bold;
+    margin-right: 24px;
+  }
+
+  .line {
+    background-color: #b8adad;
+    height: 1px;
+    width: 95.2%;
+    padding-top: 0;
+    margin-top: 125px;
+    margin-bottom: 121px;
+    margin-left: 2px;
+    margin-right: 141px;
+    flex-grow: 80%;
+  }
+
+  .work-section-image {
+    width: 85%;
+    margin-bottom: 120px;
+  }
+
+  .img {
+    height: 363px;
+    margin-top: 4px;
+    padding-top: 5px;
+    margin-left: 60px;
+  }
+
+  .work-section-text1 {
+    font-family: 'Crete Round', serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 40px;
+    line-height: 52px;
+    padding-left: 10px;
+  }
+
+  .work-section-texts {
+    margin-left: 0;
+    padding-top: 50px;
+    padding-left: 32px;
+  }
+
+  .work-section-heading {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    justify-content: center;
+    width: 87.2%;
+  }
+
+  .paragraph {
+    font-family: 'Inter', sans-serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 20px;
+    line-height: 28px;
+    padding-left: 24px;
+  }
+
+  .work-section-ul ul {
+    display: flex;
+    align-items: center;
+    border: none;
+    width: 70%;
+    margin-left: 24px;
+    padding: 60px 0 2px 20px;
+    margin-block-start: 1em;
+    margin-block-end: 1em;
+    margin-inline-start: 1em;
+    margin-inline-end: 0;
+    padding-inline-start: 0;
+  }
+
+  .work-section-ul ul li {
+    font-family: 'Inter', sans-serif;
+    font-weight: 400;
+    font-size: 14px;
+    line-height: 44px;
+    padding-right: 25px;
+    list-style: none;
+    text-align: center;
+    padding-left: 11px;
+    margin-left: 22px;
+  }
+
+  .project-section {
```

Comment:
- [ ] Please center this section because it appears more on the left side of the page. Try as much as possible to stick on [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A1471)